libbeat/docs/http-endpoint.asciidoc

```diff
@@ -33,6 +33,23 @@ current user.
 `http.named_pipe.security_descriptor`:: (Optional) Windows Security descriptor string defined in the SDDL format. Default to
 read and write permission for the current user.
 `http.pprof.enabled`:: (Optional) Enable the `/debug/pprof/` endpoints when serving HTTP. It is recommended that this is only enabled on localhost as these endpoints may leak data. Default is `false`.
+`http.pprof.block_profile_rate`:: (Optional) `block_profile_rate` controls the
+fraction of goroutine blocking events that are reported in the blocking profile
+available from `/debug/pprof/block`. The profiler aims to sample an average of
+one blocking event per rate nanoseconds spent blocked. To include every blocking
+event in the profile, pass rate = 1. To turn off profiling entirely, pass rate
+<= 0.
+`http.pprof.mem_profile_rate`:: (Optional) `mem_profile_rate` controls the
+fraction of memory allocations that are recorded and reported in the memory
+profile available from `/debug/pprof/heap`. The profiler aims to sample an
+average of one allocation per `mem_profile_rate` bytes allocated. To include
+every allocated block in the profile, set `mem_profile_rate` to 1. To turn off
+profiling entirely, set `mem_profile_rate` to 0.
+`http.pprof.mutex_profile_rate`:: (Optional) `mutex_profile_rate` controls the
+fraction of mutex contention events that are reported in the mutex profile
+available from `/debug/pprof/mutex`. On average 1/rate events are reported. The
+previous rate is returned. To turn off profiling entirely, pass rate 0. To just
```

Comment:
I need to remove that last sentence about reading from each. And mention the defaults for each.

Plus these three needed added to the reference config file.